bin/git-summary

```diff
@@ -145,7 +149,7 @@ format_authors() {
 # fetch repository age from oldest commit
 #
 repository_age() {
-  git log --reverse --pretty=oneline --format="%ar" -n 1 | LC_ALL=C sed 's/ago//'
+  git log --reverse --pretty=oneline --format="%ar" | head -n 1 | LC_ALL=C sed 's/ago//'
```

Comment:
Why revert it to an old version?

Reply:
Maybe you can confirm that it seems that git first resolves the `-n 1` and then it resolves `--reverse` what ends up in not getting the first commit but the latest.

Reply:
My tests gave me the following:

```
$  git log --reverse --pretty=oneline --format="%ar" -n 1
6 days ago

$ git log --reverse --pretty=oneline --format="%ar" | head -n 1
12 years ago
```

Reply:
Also the git-log documentation says so:

> Commit Limiting
>        ...
>        Note that these are applied before commit ordering and formatting options, such as --reverse.
> 
> -<number>, -n <number>, --max-count=<number>
>            Limit the number of commits to output.